spec/spec.ddoc

```diff
@@ -35,23 +35,6 @@ $(T style,
 SUBNAV_HEAD = $(DIVC head, $(H5 $1) $(TC p, $4, $(LINK2 $2, $3)))
 SUBNAV=$(SUBNAV_SPEC)
 SUBNAV_SPEC=
-<style type="text/css">
-body {
-  counter-reset: para h3;
-}
-p:before {
-  counter-increment: para;
-  content: counter(h3) "." counter(para) ": ";
-}
-h3:before {
-  counter-increment: h3;
-  content: "" counter(h3) ": ";
-  counter-reset: para;
-}
-h3 {
-  counter-reset: para;
-}
-</style>
```

Comment:
Please don't mix unrelated changes in the same commit. If a change in one commit negates the change in another commit, then please squash the irrelevant commit using an interactive rebase.